### ISSUE TITLE
Monthly Ticket for Quest Changes

### ISSUE THREAD

**Issue by chochem** [GTNewHorizons/GT-New-Horizons-Modpack#17882]
### Your GTNH Discord Username

chochem

### Your Pack Version

2.7.0 beta3

### Your Server

SP

### Java Version

Java 21

### Type of Server

None

### Your Expectation

placeholder

### The Reality

placeholder

### Your Proposal

placeholder

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [X] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [X] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by srdr2k3** [comment]
- [ ] UEV+ questlines do need an overhaul to accommodate for the new changes in 2.7

**Comment by chochem** [comment]
> * [ ]  UEV+ questlines do need an overhaul to accommodate for the new changes in 2.7

since Baerschen and I cant really do that, this needs some volunteer. also details lol, general statements like this are useless.
maybe make a separate ticket and start outlining things.

**Comment by srdr2k3** [comment]
> > * [ ]   UEV+ questlines do need an overhaul to accommodate for the new changes in 2.7
> 
> since Baerschen and I cant really do that, this needs some volunteer. also details lol, general statements like this are useless. maybe make a separate ticket and start outlining things.

The reason for that comment was to pique interest in the overhaul. Since the overhaul is quite new, not many players know about what the UEV, UIV and UMV progression is. I myself am a part of that portion, therefore I am not willing to write new quests or lead the effort in writing them until I reach those tiers in 2.7.0.

**Comment by chochem** [comment]
> This ticket is more focussed on small and clear problems and suggestions that don't really need any larger discussion.

you suggestion clearly is not that. Please make a separate ticket.

**Comment by FireWarrior2** [comment]
- [ ] Remove fast mode mention from XL turbine quest (2.7.0 beta 3)
![image](https://github.com/user-attachments/assets/943ebb77-67cb-4897-b0b6-519ff71d11dc)

**Comment by wlhlm** [comment]
- [ ] Quest "The Benzene Tier": The required item matching seems bugged. The quest requires an Oak and a Spruce Bonsai seed bags. I randomly opened a MV loot bag and got two Plumbilia seed bags which completed the quest somehow.

**Comment by wlhlm** [comment]
- [ ] Quest "Pressure washing!": There's a typo in the last paragraph "Mmultiblocks"

**Comment by chochem** [comment]
this is not galacticraft but amunra. and if there are bugs your main focus should be to make a ticket for the *bug itself*

**Comment by chochem** [comment]
- [ ] the chemplant blockrenderer preview might be finally fixed now, in that case the warning should be removed from the quest.

**Comment by Baerschen** [comment]
- [ ] Adjust the Tome quest for when https://github.com/GTNewHorizons/AkashicTome/pull/13 gets merged
- [ ] while at it change it back to rewarding a tome with tinker and food books (https://github.com/GTNewHorizons/AkashicTome/pull/12)

**Comment by TheKleopiNetwork** [comment]
nanoprocessor quest (AAAAAAAAAAAAAAAAAAAE4A==)
needs a description rewrite:
![image](https://github.com/user-attachments/assets/2ee851d7-4f7f-40bb-970c-74ff24a5d2d0)
at that point it would maybe also nice to have an explainer, which circuits are best to use at this point

**Comment by HoleFish** [comment]
- [ ] Hydration cell -> empty cell recipe is now an extractor recipe, not centrifuge

**Comment by feioO** [comment]
the "Advanced Crystal SoC Production" quest in UV tab has a typo. It says Cerium-dropped Lutetium instead of doped.

**Comment by chochem** [comment]
>  it would maybe also be nice to have an explainer, which circuits are best to use at this point (eg. which hv circuit is cheapest?)

that is not going to happen. the goal of the QB is to provide you with information. Not to define a meta or take over decision making/gameplay.

**Comment by chochem** [comment]
-[ ] remove quest *Processing Array Automation*. the multi is no longer craftable

**Comment by chochem** [comment]
remove PA mentions from quests:
- [ ] plantballs,
- [ ] stainless taking to long,
- [ ] empty fibre reinforced glass board

**Comment by pilchard436** [comment]
`PCB factory tier 3` quest is locked behind uiv energy hatch, but it should be locked behind uev energy hatch since we can overclock the assembler. Similarly, `thermalsink radiator` quest should be locked behind uiv energy hatch.

**Comment by TheMysticalBard** [comment]
The windmill multiblock quest incorrectly says that the rotor will be used up when there is no recipe. From testing, the rotor's durability only decreases when a recipe is in progress.

**Comment by wlhlm** [comment]
- [ ] The quest "Cooling Kanthal" gives 8 IC2 Coolant Cells as reward, its follow-up quest "Coolant" only requires 8 IC2 Coolant Cells for completion, meaning the quest is automatically checked off after completing "Cooling Kanthal" making it a bit redundant if its goal is to show the player how to make Coolant if the player doesn't have to do anything

**Comment by wlhlm** [comment]
- [ ] Quest "Venting the Heat": second paragraph needs a rephrasing. Here's my suggestion:
  > An advanced or overclocked reactor vent is much faster than a basic heat vent and can work without a water block in front of it, albeit slower. Unfortunately, you can't make those yet, but keep them in mind.

**Comment by MigaAFOMIGA** [comment]
In the beta, they made logistic pipes cheaper to start and do, but they didn't remove the requirement for other quests that are no longer used in the recipe.

**Comment by wlhlm** [comment]
- [ ] Quest "Acetic Acid" (MV) text is missing a motivation for why you'd want to make Acetic Acid, I think. Maybe a sentence along the lines of "Acetic Acid is an important part of making Ethenone on your way to Cetane-boosted Diesel"

**Comment by Impos913** [comment]
Garnet crop quest should mention the process of garnet products using nitric acid in centrifuge, lots of nice passive resources.

**Comment by Impos913** [comment]
"Light It Up!" Quest under Novice Thaumaturgy tab mentions that light sources the Arcane Lamp places can cause issues with multiblocks. This hasn't been an issue since GTNH 2.1.1.4
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/8799#issuecomment-964257534

**Comment by Sand-Master** [comment]
'The Water Dilemma' completes without having to craft all 26 water tank sidings despite it saying you need 26.

**Comment by chochem** [comment]
- [ ] mention in hologlass quest that double click on the NEI search bar is filtering the items in holoinventory

**Comment by chochem** [comment]
> 'The Water Dilemma' completes without having to craft all 26 water tank sidings despite it saying you need 26. I am on 2.7.0 official release.

hmm. that is likely caused by https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/15539

**Comment by chochem** [comment]
- [ ] fix quests for https://github.com/GTNewHorizons/GT5-Unofficial/pull/3408

**Comment by LazyFleshWasTaken** [comment]
- [ ] Add Questbook entry for the recipe filter block, probably around the sifter multiblock or ore processing.

**Comment by chochem** [comment]
> * [ ]  Add Questbook entry for the recipe filter block, probably around the sifter multiblock or ore processing.

already exists it seems :)

**Comment by chochem** [comment]
- [ ] fix typo in Recipe Filter quest
![image](https://github.com/user-attachments/assets/19849c35-50de-4f5f-b7e4-64447fd8c2de)

**Comment by SlogoSniper** [comment]
Rework UIV quest: No, it's not a Typo. Either move or delete it, if kept change wording of the quest. It currently comes before the first module of God Forge even though the item in the quest cannot be made until after the first module has already been made. The wording of the quest also implies that you must make Lossless Phonon Transfer Medium prior to making the first module of God Forge, "This incredible material displays exactly the properties you were looking for, and decide that now you can move forward with the first module of the Godforge."
![image_2024-12-22_005554271](https://github.com/user-attachments/assets/10981005-904e-4887-a659-90e2d49c5586)
![image](https://github.com/user-attachments/assets/6ebd3243-7b93-43a9-a98b-20025d4b6e21)

**Comment by Flakxo** [comment]
very low priority and maybe seen as irrelevant, but maybe consider changing the iv+ energy hatch quest to have an OR filter for higher amp hatches, since by that point you can just request them through ae2 without ever touching the regular hatch, thus not getting the completion(although its unlikely thatll happen, which is why i said low priority and maybe seen as irrelevant)

**Comment by Maykeye** [comment]
Suggestion: make it easier to delay reporting quests: so you can play a lot, then open quest book and report dozens of quest as complete. Right now several quests requiring jumping through hoops. 

First is a small one: `Crafting Time` doesn't remember you've crafted several crafting tables already. Suggestion is either to make  it remember or change crafting to retrieval.

Second: in `Your First Tool` make retrieval task for pick axe head/etc optional, as if you delay long enough you have a quest where the main goal(make pick axe, shovel, hatchet) is complete already, but the quest book doesn't remember that you had pick axe head in inventory. 

Right now even though I already reached iron tools, I'll have to remake parts in flint to advance the quest book.

**Comment by chochem** [comment]
@Maykeye this is not a suitable place for bigger discussions.

- [ ] adjust quest related to https://github.com/GTNewHorizons/GT5-Unofficial/pull/3724

**Comment by Maykeye** [comment]
> @Maykeye this is not a suitable place for bigger discussions.

OK, I didn't think it was that a big deal. I've created separate issue.

**Comment by TvL2386** [comment]
I also have a suggestion: The quest `AAAAAAAAAAAAAAAAAAAErg==` in the UV chapter called "Upgrading The Reactor" asks for 1 UV Energy Hatch, while the preview wants 16. The same goes for the ZPM quest, but not the LuV quest (this one asks for 16)

**Comment by chochem** [comment]
> I also have a suggestion: The quest `AAAAAAAAAAAAAAAAAAAErg==` in the UV chapter called "Upgrading The Reactor" asks for 1 UV Energy Hatch, while the preview wants 16. The same goes for the ZPM quest, but not the LuV quest (this one asks for 16)

dont blindly trust the preview

**Comment by NotAPenguin0** [comment]
> > I also have a suggestion: The quest `AAAAAAAAAAAAAAAAAAAErg==` in the UV chapter called "Upgrading The Reactor" asks for 1 UV Energy Hatch, while the preview wants 16. The same goes for the ZPM quest, but not the LuV quest (this one asks for 16)
> 
> dont blindly trust the preview

All other amounts for blocks in the quests are correct, it makes no sense for one of them to be wrong

**Comment by TvL2386** [comment]
> > I also have a suggestion: The quest `AAAAAAAAAAAAAAAAAAAErg==` in the UV chapter called "Upgrading The Reactor" asks for 1 UV Energy Hatch, while the preview wants 16. The same goes for the ZPM quest, but not the LuV quest (this one asks for 16)
> 
> dont blindly trust the preview

I don't blindly trust the preview, which is why I've asked on discord, but I've been told I should know the drill by now. So what is it now? I don't understand... 
The quest says it can handle recipes up to 640M startup, maybe the quest (and ZPM quest) should then at least tell me in advance how much EU a hatch adds, like the LuV quest

**Comment by chochem** [comment]
> > > I also have a suggestion: The quest `AAAAAAAAAAAAAAAAAAAErg==` in the UV chapter called "Upgrading The Reactor" asks for 1 UV Energy Hatch, while the preview wants 16. The same goes for the ZPM quest, but not the LuV quest (this one asks for 16)
> > 
> > 
> > dont blindly trust the preview
> 
> All other amounts for blocks in the quests are correct, it makes no sense for one of them to be wrong

how many you need is recipe dependent. 16 is almost certainly not the right number.

**Comment by bluhbipo** [comment]
> > > > I also have a suggestion: The quest `AAAAAAAAAAAAAAAAAAAErg==` in the UV chapter called "Upgrading The Reactor" asks for 1 UV Energy Hatch, while the preview wants 16. The same goes for the ZPM quest, but not the LuV quest (this one asks for 16)
> > > 
> > > 
> > > dont blindly trust the preview
> > 
> > 
> > All other amounts for blocks in the quests are correct, it makes no sense for one of them to be wrong
> 
> how many you need is recipe dependent. 16 is almost certainly not the right number.

the neutronium recipe for example needs all 16 hatches

**Comment by chochem** [comment]
its optional

**Comment by bluhbipo** [comment]
> its optional

ok then how about we make it 13 energy hatches which is the minimum for americium?

**Comment by cecilialau6776** [comment]
Quest `AAAAAAAAAAAAAAAAAAADNg==` Space Race -> Oxygen Gear` should have quest `AAAAAAAAAAAAAAAAAAAAUA==` Multiblock Goals -> Compress All The Things as a prerequisite (current version 2.7.0)

**Comment by chochem** [comment]
- [ ] fix flower power quest. the seed complete without checking the species.

**Comment by chochem** [comment]
- [ ] fix outdated NEI hint in quest 'Pipes, Pipes Everywhere' as NEI search was changed in 2.7.0

**Comment by cecilialau6776** [comment]
I suggest that quest `AAAAAAAAAAAAAAAAAAAFEg==` (Applied Energistics -> Wireless Setup) change pronoun from "himself" to "theirself" (or "themselves" or "themself" or another gender-neutral pronoun) in "... cannot exclude himself any rights."

**Comment by KleinGrrmpf** [comment]
In the quest "Separate Those Last Specks" in the Monaline, it says that you get some Chlorobenzene when centrifuging, but you actually get Chloromethane back.

**Comment by cecilialau6776** [comment]
Quest `b_Ll4zvSRfCMgmqiBa4Xkw==` (Mutliblock Goals -> Don't Lose That Multiblock Miner!) misspells "Sensor" as "Senor" in "Unlike the GregTech Senor Kit..."

**Comment by wlhlm** [comment]
- [ ] Quest "Let's Get Crackin!" Adjust Cupronickel Coil Block amount from 16 to 20 (4 are required for the controller block).

**Comment by eigenraven** [comment]
- [ ] Describe that twilight forest raven feathers can be obtained by scaring off ravens by running at them in addition to killing them, see https://github.com/GTNewHorizons/twilightforest/pull/102 (20% drop probability, 20 second cooldown, idk if we want to be this precise in the qb as we might want to rebalance the numbers later)

**Comment by wlhlm** [comment]
- [ ] Quest "Silky Jewel" Adjust description to consider GT ores (Silk Touch is now usable on them).
- [ ] Quest "Where'd I Put That Amber?" Remove mention of GT Tools.

**Comment by serenibyss** [comment]
New ae2 parts to consider for quests:
- [ ] [Fake Crafting Card](https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/pull/622)
- [ ] [Equal Distribution Card](https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/pull/655)
- [ ] [Overflow Void Card](https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/pull/655)
- [ ] [Superliminal Acceleration Card](https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/pull/617)
- [ ] [Neutronium Energy Anchor](https://github.com/GTNewHorizons/Applied-Energistics-2-Unofficial/pull/624)

**Comment by Baerschen** [comment]
Oversight of mentions of making PAs 
![Image](https://github.com/user-attachments/assets/cb27ab2f-7473-4071-8365-6b481fd9e08c)

**Comment by Dream-Master** [comment]
2x and 4x EBF need a new icon since double and quadrupple ingots are gone :(

![Image](https://github.com/user-attachments/assets/2000d749-21e6-47fc-bd82-6a562094df95)

**Comment by Vsevolod090** [comment]
I suggest adding quest about certus quartz tank, connected to the quarzite vein quest

**Comment by Dream-Master** [comment]
> 2x and 4x EBF need a new icon since double and quadrupple ingots are gone :(
> 
> ![Image](https://github.com/user-attachments/assets/2000d749-21e6-47fc-bd82-6a562094df95)

fixed btw

**Comment by EthanGraber** [comment]
![Image](https://github.com/user-attachments/assets/d9bcfb9a-59dd-4ae1-b9cd-5ecdf6bd3086)

Very slight typo in the EBF Multiblock quest (mufffler)

**Comment by barsikus007** [comment]
wrong items in thermal boiler quest
AAAAAAAAAAAAAAAAAAAH6w==

![Image](https://github.com/user-attachments/assets/ee72afd1-809c-47e2-89ee-d86757a0e746)

**Comment by Lxran** [comment]
Applied Energistics tab, Dense Boys (AAAAAAAAAAAAAAAAAAAFEw==) quest mentions the Dense Energy Cell can hold 1.6 million units when in reality the Dense Energy Cell can hold *16* million AE.

![Image](https://github.com/user-attachments/assets/0adb9930-15fb-4879-817a-c2847c7016d3)

**Comment by Celarious** [comment]
Minor typo in the Steel Chest BBB quest `V4jinTYRSueIrEqn90dRzQ==`

Also made a PR for this because it's a quick and easy fix https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/18977

![Image](https://github.com/user-attachments/assets/a0d92199-f235-4027-9a09-c8e81c5171cb)

**Comment by Kyramesh** [comment]
![Image](https://github.com/user-attachments/assets/a2c19939-1375-4ba3-9188-aaabea10a0d9)

The qb is accurate here. 500B per module is wildly off. Mk3 Space Assemblers run 64 parallel @uxv. 1 uxv is 537M. 534M x 64 parallel = 34.2B. A whole SE running the max of 24 of them is about 820B.

**Comment by WarriorAvi4246** [comment]
Discord Username: warrioravi
Pack Version: Nightly 867

The "Gravel: The Gathering" Quest in the Stone Age chapter still mentions flint for more tools despite the change to starting with wooden tools. I think this is possibly an oversight? Thanks.

**Comment by WarriorAvi4246** [comment]
Discord Username: warrioravi
Pack Version: Nightly 867

The "Your First Tool" quest in the Stone Age Chapter says swords are unobtainable until metals like iron or bronze is available, but a quest to make a Tinker's flint broadsword was added. Additionally it mentions GregTech swords which were removed. Thanks!

![Image](https://github.com/user-attachments/assets/f605e6a7-3db5-48e8-bd94-52c9990b015b)

**Comment by Samario1** [comment]
![Image](https://github.com/user-attachments/assets/4b7ce313-7761-4c88-aa1b-962bc7cf1614)
"Your Oil is Too Crude" quest has a redundant OR condition on it, since you have to make a pump to be able to unlock the Black Gold quest anyway so there's never a situation where you could have the oil quest done but not the pump quest

**Comment by JustAlkaid** [comment]
**Discord Name**: JustAlkaid

**Pack Version**: 2.7.0

**Related Quest**: AAAAAAAAAAAAAAAAAAALlQ== & KHL_yx4XTKOtsHHZ60ZZpg==

Recipes of Naqfuel MkV and MkVI have been buffed. Naqfuel MkV doesn't need Spacetime anymore and Naqfuel MkVI need Condensed Raw Stellar Plasma Mixture instead of White Dwarf Matter.

**Comment by JustAlkaid** [comment]
**Discord Name**: JustAlkaid

**Pack Version**: 2.7.0

**Related Quest**: _O6stWM3RR-qJdcvs0i-Mw==

The Generated EU formula of antimatter generator is wrong. 1L antimatter provide 1e12 EU instead of only 1e9, which the wrong formula claimed.

**Comment by JustAlkaid** [comment]
**Discord Name**: JustAlkaid

**Pack Version**: 2.7.0

**Related Quest**: _AAAAAAAAAAAAAAAAAAAKqw==

Active Transformer would NOT explode any more.

**Comment by kotkowski** [comment]
Discord Name: kotkowski

Pack Version: 2.7.0

Related Quest: AAAAAAAAAAAAAAAAAAAHFA==

HV Quest Teflon Don wording seemingly reccomends chemplant with sand as a source of pre-moon fluorine, though recipe is locked behind EV (Tungstensteel Casings)

Would be nice to remove that line or mention it at another paragraph, as it''s bit misleading.

![Image](https://github.com/user-attachments/assets/01473d78-f548-495c-b112-d18c1e72e15b)

![Image](https://github.com/user-attachments/assets/8a1a0358-9fa6-4589-9c16-aaa7b08ce2df)

**Comment by Kynake** [comment]
Discord name: kynake

Pack version: Nightly 907

Related Quest: AAAAAAAAAAAAAAAAAAAHwg==

Now that the Architect's Glowbrush has been added to ArchitectureCraft the quest 'More Like Ouch-i-Tect' should be updated to reward it together with the other tools, as well as an updated description that explains how to use it.

**Comment by ALongStringOfNumbers** [comment]
The quest titled `Automating Power On/Off` still mentions that non-GT++ multis will void when full, even though most of them now have Voiding Protection.

Quest ID: AAAAAAAAAAAAAAAAAAAGdg==

Nightly 903

**Comment by leumasme** [comment]
- [ ] https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/19473 

TLDR The quest before steam forge hammer makes the player craft two anvils, even though only one is useful. This leads to the quest feeling more like a hindrance as making just one anvil and then the forge hammer will leave you behind in quests, especially so because the forge hammer is a craft quest.

**Comment by Celarious** [comment]
> The quest description for "Are You Prepared For HV?" says "Chrome can be centrifuged out of ruby dust", but it's actually electrolyzed.
> 
> Version: 2.7.3 Quest ID: AAAAAAAAAAAAAAAAAAAC7Q==

Checked and it seems there is a recipe to centrifuge chrome out of impure ruby dust (CrAl2O3), at an 11.11% chance. So the quest is technically right, even if it's less efficient than electrolysis

**Comment by JustAlkaid** [comment]
**Discord Name**： Just Alkaid

**Quest Name**：Sending the Power Back

**Quest ID**：AAAAAAAAAAAAAAAAAAAEeQ==

**Quest Tier**：HV

160-93=**97** instead of **57** so the loss is **97**EU per Ampere

**Comment by OmdaCZ** [comment]
Discord Name： ondrejmenzel

Quest Name：Waterproof Tech, Cribs

Quest ID：1FFkSYHcQPO_6U185CKBiw== , 4pCol8RvR3G0pS2zVqpY1Q==

Quest Tier：UIV/Applied Energistics, Applied Energistics

The Waterproof Tech quest is incorrectly gated by spacetime quest, which crafting input buffers do not need anymore, similar issue is in the Cribs quest, where it says, that Crafting Input Proxy is UV-tier and Crafting Input Buffer is UIV-tier, however their tiers have been changed to UIV (proxy) and UEV (buffer).

**Comment by PaulMPomichter** [comment]
Discord Name： Burmecia

Quest Name：Platinum Processing, Step 3 | Performing Chemical Experiments at HV Level

Quest ID：AAAAAAAAAAAAAAAAAAAI-Q== | AAAAAAAAAAAAAAAAAAADrw==

Quest Tier：Tier 4 - EV | Tier 3 - HV

Platline quest required 10 cells of Diluted Sulfuric Acid, but in current 2.8 builds the recipe is fixed to give Hydrochloric Acid instead.

HV Chem reactor quest requires a redundant HV CR single block, when you can make LCR as soon as you get HV hulls.

**Comment by AdityaVG13** [comment]
Discord Name: AdtiyaG13

Quest Name: Get that Stone

Quest ID: AAAAAAAAAAAAAAAAAAAACA==

Quest Tier: And So, It Begins

Problem: Text says "Look at you, with your shiny flint pickaxe!" 

Solution: Flint GT Tools are no longer part of the questline, needs to be changed to say "wood tools" presumably

**Comment by Quarri6343** [comment]
Discord Name: Quarri6343

Quest Name: You Are Not Prepared!!!

Quest ID: AAAAAAAAAAAAAAAAAAAL1w==

Quest Tier: Multiblock Goals

Problem: Quest rewards 10 Aluminum Oreberries which have no use since early game Aluminum has been replaced by Zinc

Nightly 964

**Comment by dibbydoda** [comment]
Discord Name: dibbydoda

Quest Name: Cooking for Blockheads

Quest ID: AAAAAAAAAAAAAAAAAAALTw==

Quest Tier: Fishing Farming Cooking

**Once** https://github.com/GTNewHorizons/CookingForBlockheads/pull/54 **is merged**: add mention that ingredients can be stored in JABBA Barrels or Storage Drawers connected to the multiblock kitchen.

**Comment by YannickMG** [comment]
Discord Name: Yannick

Quest Name: Wood Tar

Quest ID: AAAAAAAAAAAAAAAAAAAMIg==

Quest Tier: Bio for the Masses

Problem: The Wood Tar to Benzene quest is said to be LV, but it is an MV recipe.

Solution: Drop all mentions of overclocking and using LV machines.

**Comment by 2ndDerivative** [comment]
I'm just gonna link [https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/19586](this) here

**Comment by sweetcornwhiskey** [comment]
Discord Name: sweetcornwhiskey

Quest Name: Storing the Fluids; Electrolyzing Water; Desulfurizing the Oil; Dealing with H2S

Quest ID: AAAAAAAAAAAAAAAAAAAGZQ== AAAAAAAAAAAAAAAAAAAGZg== AAAAAAAAAAAAAAAAAAAGZw== AAAAAAAAAAAAAAAAAAAGaA==

Quest Tier: Basic Automation

Problem: Super tanks are required to complete these quests, which require MV tier components. However, it is clear that PR automation is intended to be an LV tier automation solution.

Solution: These quests should require LV tanks instead of super tanks, and the text in the quests should replace all instances of "super tank" with "tank" to fit the required voltage tier.

**Comment by LazyFleshWasTaken** [comment]
Add quest completion reward for HV chem reactor for the Large chemical reactor, as you can make PTFE in MV, and not have to make the HV CR at all.

**Comment by HFPTetraUro** [comment]
I messed up my issue mentions in my PR but #19803 closes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/19609 and the cactus/sugar charcoal bullet

**Comment by combusterf** [comment]
I'd like to add https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/20317 as a suggestion to separate QFT shieldings from manipulators

**Comment by Niasty** [comment]
Quest Ids: AAAAAAAAAAAAAAAAAAADKA== ; AAAAAAAAAAAAAAAAAAADow== ; AAAAAAAAAAAAAAAAAAADog==
Quest tier: HV, MV, LV

Add MV miner quest completion as a reward for doing the HV miner quest
same with the LV miner for doing the MV miner quest

they're needed to unlock the firestone and long distance item pipeline quests, and it would be nice to have them unlocked even if you only start auto mining in hv

**Comment by GiribaldiE** [comment]
Large sifter multiblock quest gives Vanilla Diamond ores as the quest reward. Thought it wasn't intentional as the emerald ore one is gregtech version
Quest ID is AAAAAAAAAAAAAAAAAAAFjA==

<img width="592" height="197" alt="Image" src="https://github.com/user-attachments/assets/da6956b9-2c27-4fea-be7e-e0f3827ff46f" />

**Comment by TheRealShadoww** [comment]
I think the UMV quest should be locked behind Forge of the God, it kinda odd that you can reach UMV and only completing 4 or 5 quests in UIV.

**Comment by HFPTetraUro** [comment]
> Quest Ids: AAAAAAAAAAAAAAAAAAADKA== ; AAAAAAAAAAAAAAAAAAADow== ; AAAAAAAAAAAAAAAAAAADog== Quest tier: HV, MV, LV
> 
> Add MV miner quest completion as a reward for doing the HV miner quest same with the LV miner for doing the MV miner quest
> 
> they're needed to unlock the firestone and long distance item pipeline quests, and it would be nice to have them unlocked even if you only start auto mining in hv

did this in #20502

**Comment by Star-Tube** [comment]
The quest "Really (Really!) Dark Magic" should complete upon picking up a crimson rite regardless of whether the quest "Secrets" has been completed.

This is primarily relevant to people who use the alternate recipe for the crimson rite rather than fight the crimson guard.

**Comment by HFPTetraUro** [comment]
anyone know how to create trigger quests for line skips? Can you do it with the ingame quest book editor? I didn't find a way to view/create quests with no quest line, or do you have to do it manually?

**Comment by Kynake** [comment]
Almost no one seems to know that the Dislocator Inhibitor from Draconic Evolution was backported. To give it more visibilty, I think it should get it's own quest in MV, since it is very useful for automations that require items on the ground (Especially Botania)

Here's a sketch of where it could be located:
<img width="1274" height="792" alt="Image" src="https://github.com/user-attachments/assets/be9a28f5-4114-4662-b643-af3d4afe42b2" />

**Comment by KleinGrrmpf** [comment]
<img width="2418" height="523" alt="Image" src="https://github.com/user-attachments/assets/ad8e47ca-605d-49ed-a583-ade781da48ff" />

Either these two are switched, or there is a different case for Input/Output Hatches. Because you need to set it to "Export" when you want to send something FROM and Output Hatch, and to "Import" when you want to receive something INTO an Input Hatch.

EDIT: Quest ID: AAAAAAAAAAAAAAAAAAAGVQ==

**Comment by someonelucas** [comment]
The nine fuel rod quests in the "How to Generate Power" tab have item placeholders as icons, tasks, and sometimes as rewards. 

<img width="231" height="248" alt="Image" src="https://github.com/user-attachments/assets/bb7e5dab-4dcc-4df1-9ca4-ea5b3addc1f4" />

Version: 2.8.0-beta-2
Quest IDs: AAAAAAAAAAAAAAAAAAAAwQ==, AAAAAAAAAAAAAAAAAAAHaw==, AAAAAAAAAAAAAAAAAAAHbA==, AAAAAAAAAAAAAAAAAAAAvw==, AAAAAAAAAAAAAAAAAAAHaA==, AAAAAAAAAAAAAAAAAAAAxg==, AAAAAAAAAAAAAAAAAAAB6g==, AAAAAAAAAAAAAAAAAAAAwA==, AAAAAAAAAAAAAAAAAAAHaQ==

**Comment by UltraProdigy** [comment]
> The nine fuel rod quests in the "How to Generate Power" tab have item placeholders as icons, tasks, and sometimes as rewards. 

This is fixed in latest

**Comment by tomprince** [comment]
It would be nice if the quests for iron+ chests/barrels/tanks all accepted an equivalent upgrade item instead. If I have a full container, I don't want to have to empty and break it just to get the question completion.

**Comment by PPKFS** [comment]
You Need Them All Part 2 is not possible to be completed in the Stone Age, despite being present in the Stone Age chapter.

Thaumaturge Backpack requires Amber (Twilight Forest, so LV) and the Apothecary Backpack requires Instant Healing II Potions (Brewing stand so MV, or Skeleton Druid drops for LV). 

I also don't believe you can get packed ice before LV for the Iceman's Backpack.

Proposal: moving these 3 backpacks to either the getting around tab or to LV somewhere and then combining the two You Need Them All quests into one.

**Comment by feioO** [comment]
Your GTNH Discord Username
FeioO

Your Pack Version
2.8.0beta4

Quest ID: AAAAAAAAAAAAAAAAAAADXg==
Mentions battlegear slots for the arrows that have been removed in 2.8

**Comment by SST-03** [comment]
version: 2.8.0beta2
Quest ID: AAAAAAAAAAAAAAAAAAAM0w== (1) / AAAAAAAAAAAAAAAAAAAKUg== (2) / AAAAAAAAAAAAAAAAAAAKVA== (3)

Making Space Elevator(1) can skip T7/T8 Rocket (2)(3). Also you just need 32x T7Plate to make Space Elevator, so maybe there should be a skip to unlock Space Elevator event?

**Comment by tomprince** [comment]
version: 2.8.0beta4

I'm surprised there is no ender fluid conduit quest. They and the item conduits seem the most useful of the conduits.

**Comment by SST-03** [comment]
version: 2.8.0beta2
please make a questline to talk about SpaceProjectManager.

**Comment by glacialvitality** [comment]
version: 2.8.0 beta 4 (daily 131)
please add a trigger skip quest for the monazite/bastanite line

**Comment by KleinGrrmpf** [comment]
What about making the quests in UV for the bacteria only require the final item (like the General Purpose Fermenting Bacteria) instead of all the intermediate steps? I already made all the culture before the quest was unlocked and that would require me to remake the specific plasmids etc. So just a bunch of work for something that I already have

**Comment by LewisSaber** [comment]
version: 2.8.0 beta 3
Quest ID:YqImjtIqSZCc-Mlri03SEA==

personal healing nanobooster quest is in ZPM , while It can only be made in UV
(one of components need Titansteel, alloy made in UV ABS(you cant skip tier in abs so it cant be made in zpm like other UV recipes))
